### ISSUE TITLE
Pagination has no semantics of currently active page

### ISSUE THREAD

**Issue by petermakowski** [canonical-web-and-design/react-components#732]
Pagination has no semantics for indication of currently active page.
This makes for a bad accessibility and difficult to test using testing-library.

**Comment by bartaz** [comment]
Vanilla pagination supports `aria-current="page"`, so it should be implemented into React component as well.

I guess that's where it can be fixed:

https://github.com/canonical-web-and-design/react-components/blob/01c63a24d877e699eb3a37ae18d00b02b4e884d7/src/components/PaginationItem/PaginationItem.tsx#L26-L33
